x/wasmplus/keeper/querier_test.go

```diff
@@ -36,24 +37,50 @@ func TestQueryInactiveContracts(t *testing.T) {
 	}
 }
 
-func TestQueryIsInactiveContract(t *testing.T) {
+func TestQueryInactiveContract(t *testing.T) {
 	ctx, keepers := CreateTestInput(t, false, AvailableCapabilities)
 	keeper := keepers.WasmKeeper
 
 	example := InstantiateHackatomExampleContract(t, ctx, keepers)
-
+	contractAddr := example.Contract
 	q := Querier(keeper)
-	rq := types.QueryInactiveContractRequest{Address: example.Contract.String()}
-	res, err := q.InactiveContract(sdk.WrapSDKContext(ctx), &rq)
-	require.NoError(t, err)
-	require.False(t, res.Inactivated)
 
-	// set inactive
-	err = keeper.deactivateContract(ctx, example.Contract)
-	require.NoError(t, err)
```

Comment:
I consider this to be related to the environment configuration. Writing it here would be more appropriate than including it within the loop.

Reply:
Thanks. I fixed it!